File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/util/DeleteObject.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class DeleteObject

    Public Shared Sub Run()

        Dim objectName As String = "CostCenter"
        Dim primaryKey As String = "TST"
        Dim deleteObjectHttpBinding As DeleteObjectHttpBinding = PaceClient.getDeleteObjectHttpBinding()
        Try
            deleteObjectHttpBinding.deleteObject(objectName, primaryKey)
            Console.WriteLine(objectName + " " + primaryKey + " deleted")
        Catch ex As Exception
            Console.WriteLine("Delete Object failed : " + ex.Message)
        End Try

    End Sub

End Class

File: docs/legacy(old PACE)/C#/PaceAPISample/util/DeleteObject.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using efipaceservices;

namespace Pace_Web_Service_SDK.util
{
    class DeleteObject
    {
        public static void Run()
        {
            String objectName = "Job";
            String primaryKey = SDK.readStringInputFromTheUser("Please provide id of the Job to be deleted");
            DeleteObjectPortType deleteObjectHttpBinding = PaceClient.getDeleteObjectHttpBinding();
            try
            {
                deleteObjectHttpBinding.deleteObject(objectName, primaryKey);
                Console.WriteLine(objectName + " " + primaryKey + " deleted");
            }
            catch (Exception e)
            {
                Console.WriteLine("Delete Object failed : " + e.Message);
            }
        }
    }
}
